packages/app/src/components/profile/ButtonNav.tsx

```diff
@@ -1,25 +1,54 @@
 import React, { ReactElement } from 'react';
-import { View, StyleSheet } from 'react-native';
+import { View, StyleSheet, Text } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 import { ImageCard, Title, sizes } from '../UI';
+import { useNavigation } from '@react-navigation/native';
+import { Profile_me$key } from './__generated__/Profile_me.graphql';
+import { useFragment, graphql } from 'react-relay';
+import { ButtonNav_me$key } from './__generated__/ButtonNav_me.graphql';
 
-type Props = {};
+type Props = {
+    me: ButtonNav_me$key;
+};
 
 export const ButtonNav = (props: Props): ReactElement => {
-    const cards = ['Challenges', 'Cards'];
+    const me = useFragment<ButtonNav_me$key>(
+        graphql`
+            fragment ButtonNav_me on User {
+                ...UserChallengesList_me
+                ...UserChallengeDetailList_me
+            }
+        `,
+        props.me
+    );
+    const cards = [
+        { Challenges: 'UserChallengesList' },
+        { Cards: 'UserChallengeDetailList' }
```

Comment:
The cards screen should open to a different `UserChallengeCardsList` component. This component should query for the `challengeCards` field on `me` and show a swipeable list of all the cards